services/apps/alcs/src/alcs/notice-of-intent-decision/notice-of-intent-decision-component/notice-of-intent-decision-component.service.spec.ts

```diff
@@ -0,0 +1,234 @@
+import { ServiceValidationException } from '@app/common/exceptions/base.exception';
+import { createMock, DeepMocked } from '@golevelup/nestjs-testing';
+import { Test, TestingModule } from '@nestjs/testing';
+import { getRepositoryToken } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { CreateNoticeOfIntentDecisionComponentDto } from './notice-of-intent-decision-component.dto';
+import { NoticeOfIntentDecisionComponent } from './notice-of-intent-decision-component.entity';
+import { NoticeOfIntentDecisionComponentService } from './notice-of-intent-decision-component.service';
+
+describe('ApplicationDecisionComponentService', () => {
```

Comment:
NoticeOfIntentDecisionComponentService. And in general application->noticeOfIntent